python/tidy/servo_tidy/licenseck.py

```diff
@@ -7,97 +7,14 @@
 # option. This file may not be copied, modified, or distributed
 # except according to those terms.
 
+mpl = "This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/"
```

Comment:
Style Nit: Also, the name could be in uppercase, since it's a constant (even though it doesn't make any difference in python).